src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -3194,8 +3194,114 @@ def test_sql_managed_db_short_retention(self, resource_group, resource_group_loc
                      self.check('retentionDays', '{retention_days_dec}')])
 
 
-class SqlManagedInstanceRestoreDeletedDbScenarioTest(ScenarioTest):
+class SqlManagedInstanceDbLongTermRetentionScenarioTest(ScenarioTest):
+    @record_only()
+    def test_sql_managed_db_long_term_retention(
+            self):
+
+        self.kwargs.update({
+            'rg': 'clitestxj6awmetud',
+            'loc': 'westus',
+            'managed_instance_name': 'ayang-ltr-mi',
```

Comment:
Because you are using an existing managed instance, one day someone else takes over this part and it will be hard to re-run the test. 
And CLI live run test will skipped.
Is this acceptable for you?

Reply:
Yes; I've spoken to @arrownj about this scenario.  In fact I prefer not to create new resources every time.

Reply:
spoke on teams